engine/src/main/java/org/terasology/logic/players/DebugControlSystem.java

```diff
@@ -74,24 +78,45 @@ public void onHideHUD(HideHUDButton event, EntityRef entity) {
         }
     }
 
+    /**
+     * Increases view distance upon receiving an increase view distance event.
+     * @param button The button or key pressed to increase view distance.
+     * @param entity The entity corresponding to event.
+     */
     @ReceiveEvent(components = ClientComponent.class)
     public void onIncreaseViewDistance(IncreaseViewDistanceButton button, EntityRef entity) {
         int viewDistance = config.getRendering().getViewDistance().getIndex();
         int maxViewDistance = ViewDistance.values().length - 1;
-
+        
+        //Ensuring that the view distance does not exceed its maximum value.
         if (viewDistance != maxViewDistance) {
-            config.getRendering().setViewDistance(ViewDistance.forIndex((viewDistance + 1)));
+            ViewDistance greaterViewDistance = ViewDistance.forIndex(viewDistance + 1);
+            config.getRendering().setViewDistance(greaterViewDistance);
```

Comment:
Maybe move the actual setting of the view distance below the logging? That way if anything fails during the view distance change the chat panel should show what's about to happen, even if briefly (for the case where a player might have hit the key accidentally and didn't realize how they crashed the game)

You can tweak the notification message slightly to something like "Increasing view distance to:"